src/reactingFlow.cpp

```diff
@@ -1268,11 +1307,58 @@ void ReactingFlow::step() {
     Tn_next_gf_.GetTrueDofs(Tn_next_);
   }
 
-  // explicit filter species (is this necessary?)
+  // explicit filter species
+  // Add if found to be necessary
+
+  if (operator_split_) {
+    /// PART II: time-splitting of reaction
+
+    // Notes:
+    // 1) store phi^{n} fields in buffer so substeps can use {n}
```

Comment:
Update comments (b/c they are either not up to date or confusing or both) and clean out commented out code.